SetupScript.sh

```diff
@@ -125,7 +125,7 @@ mv /opt/Epsilon-Raspberry/QT_CFLAGS_DBUS.patch /home/pi/qt5
 	-prefix /home/pi/qt5 |& tee "output_configure.txt")
 (cd /home/pi/qt5/qtbase && make |& tee "output_make.txt")
 (cd /home/pi/qt5/qtbase && make install |& tee "output_make_install.txt")
-qtchooser -install qt55 /home/pi/qt5/bin/qmake
+ln -s /home/pi/qt5/bin/qmake /usr/bin/qmake
```

Comment:
I think you have to remove the old link before doing this. 

`rm /usr/bin/qmake`

Reply:
In general it's good practice to not remove stuff in scripts that the script did not make itself. Making a backup is better.

`mv /usr/bin/qmake /usr/bin/qmake.bak`